bundles/org.openhab.binding.feed/src/main/java/org/openhab/binding/feed/internal/handler/FeedHandler.java

```diff
@@ -100,6 +104,17 @@ private void checkConfiguration() {
                     DEFAULT_REFRESH_TIME, e.getMessage());
             refreshTime = DEFAULT_REFRESH_TIME;
         }
+
+        try {
```

Comment:
In general it is better to use a configuration class. You don't have to deal with the type casting or default values then. 

https://github.com/openhab/openhab-addons/blob/e4b2e1c88da65664b2d4110c06caae532bccefdb/bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/DeconzBaseThingHandler.java#L55
https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/ThingConfig.java



https://github.com/openhab/openhab-addons/blob/e4b2e1c88da65664b2d4110c06caae532bccefdb/bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/DeconzBaseThingHandler.java#L181